src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -10975,7 +10941,6 @@ void VRFTerminalUnitEquipment::CalcVRFIUVariableTeTc(EnergyPlusData &state,
     // Get the equipment/zone index corresponding to the VRFTU
     CoolCoilNum = this->CoolCoilIndex;
     HeatCoilNum = this->HeatCoilIndex;
-    ZoneIndex = this->ZoneNum;
```

Comment:
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:10978]:(style),[unreadVariable],Variable 'ZoneIndex' is assigned a value that is never used.